packages/dev/docs/pages/releases/2023-12-18.mdx

```diff
@@ -59,14 +59,18 @@ We had to temporarily remove nested column support in Tables due to some last mi
 * Add support for onScroll event to RAC collection components - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5534)
 * Add hover state to `TableColumnHeader` when 'allowSorting' is true - [@sookmax](https://github.com/sookmax) - [PR](https://github.com/adobe/react-spectrum/pull/5471)
 * Use owner document in FocusScope - [@slye-stripe](https://github.com/slye-stripe) - [PR](https://github.com/adobe/react-spectrum/pull/5449)
+* Change RAC style merge orders [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/5506)
```

Comment:
I left this one out intentionally, it's covered by `RAC support overriding default styles in overlay components`